app/controllers/performance_dashboards/overview_controller.rb

```diff
@@ -82,5 +95,39 @@ def details
     private def default_comparison_pattern
       PerformanceDashboards::Overview.comparison_patterns.values.first
     end
+
+    private def render_pdf
+      file_name = "Performance Overview #{DateTime.current.to_s(:db)}"
```

Comment:
Nice!  I haven't seen this `.to_s(:db)` before.  Way cleaner than our usual.

Reply:
Oddly, the end result when downloaded appears to be `download.pdf` so I guess it's not working yet.

Reply:
yeah, the file name mismatch is a result of the js hack to "download" the file from an ajax response. It will be replaced with a proper rendering pipleline.